### PR TITLE
 Add Con-Bench helm chart to the Vald charts repository

### PR DESCRIPTION
**Pull request by vankichi**
### Description

SSIA 

### Versions

- Go Version: 1.22.0
- Docker Version: 20.10.8
- Kubernetes Version: v1.29.2
- NGT Version: 2.1.6

### Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/main/CONTRIBUTING.md) document and completed [our CLA agreement](https://cla-assistant.io/vdaas/vald).
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?

### CODE REVIEW

**Review by hlts2 — DISMISSED** [review]
LGTM!

File: Makefile.d/helm.mk [hunk 1]
```diff
@@ -45,6 +45,14 @@ helm/package/vald-helm-operator: \
 	helm/schema/crd/vald-helm-operator
 	helm package charts/vald-helm-operator
 
+.PHONY: helm/package/vald-benchmark-operator
+## packaging Helm chart for vald-helm-operator
+helm/package/vald-benchmark-operator: \
+	helm/schema/crd/vald-benchmark-job \
+	helm/schema/crd/vald-benchmark-scenario \
+	helm/schema/crd/vald-benchmark-operator
+	helm package charts/vald-benchmark-operator
```
Comment (ykadowak) [inline Makefile.d/helm.mk:54]:
```suggestion
	helm/schema/crd/vald-benchmark-operator \
	helm package charts/vald-benchmark-operator
```

Comment (ykadowak) [reply Makefile.d/helm.mk:54]: Ah.. forget it

File: charts/vald-benchmark-operator/README.md [hunk 2]
```diff
@@ -1,24 +1,25 @@
-# vald-benchmark-operator
+# Vald Benchmark Operator
 
-![Version: v1.7.5](https://img.shields.io/badge/Version-v1.7.5-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.16.0](https://img.shields.io/badge/AppVersion-1.16.0-informational?style=flat-square)
+This is a Helm chart to install `vald-benchmark-operator`.
 
-A benchmark operator for benchmarking the Vald cluster.
+Current chart version is `v1.7.5`
```
Comment (ykadowak) [inline charts/vald-benchmark-operator/README.md:5]: This should be `v1.7.11`?

File: charts/vald-benchmark-operator/Chart.yaml [hunk 3]
```diff
@@ -32,7 +32,7 @@ appVersion: "1.16.0"
 
 apiVersion: v2
 name: vald-benchmark-operator
-version: v1.7.5
+version: v1.7.11
```
Comment (ykadowak) [inline charts/vald-benchmark-operator/Chart.yaml:35]: version needs to be updated by ci like below
https://github.com/vdaas/vald/blob/9b088e3db26724f4a6f37e45e1ab6b5633f8982b/.github/workflows/_release-pr.yml#L77

**Pull request closed without merging**